bowtie/_cli.py

```diff
@@ -852,9 +852,10 @@ def _validation_results_table_in_markdown(
         rows_data.append(row_data)
 
     for idx, row_data in enumerate(rows_data):
-        final_content += f"### {idx+1}. Schema:\n {row_data[0]}\n\n"
+        final_content += f"### {idx+1}. Schema:\n ```\n{row_data[0]}\n```\n\n"
```

Comment:
Should we add a language tag for JSON as well while we're here?

Thanks a lot for the fix!

Reply:
Added the json suggestion